packages/teleport/src/Discover/Kubernetes/HelmChart/HelmChart.tsx

```diff
@@ -46,12 +45,14 @@ import type { AgentStepProps } from '../../types';
 import type { JoinToken } from 'teleport/services/joinToken';
 import type { AgentMeta, KubeMeta } from 'teleport/Discover/useDiscover';
 import type { Kube } from 'teleport/services/kube';
+import type { Poll } from 'teleport/Discover/Shared/CommandWithTimer';
 
 export default function Container(
   props: AgentStepProps & { runJoinTokenPromise?: boolean }
 ) {
   const [namespace, setNamespace] = useState('');
   const [clusterName, setClusterName] = useState('');
+  const [showHelmChart, setShowHelmChart] = useState(props.runJoinTokenPromise);
```

Comment:
Alternatively, instead of exporting a Container as default export you may consider this naming convention

```ts
// initializes state hooks and inject it to the view
export const HelmChart = (props) => {
  const state = useHelmChart()
  return <HelmChartView {...state} />
}

// renders the state passed via props
export const HelmChartView = props => {
}
```